oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/BranchCommitGCTest.java

```diff
@@ -82,6 +85,19 @@ public static java.util.Collection<Object[]> params() throws IOException {
         for (Object[] fixture : AbstractDocumentStoreTest.fixtures()) {
             DocumentStoreFixture f = (DocumentStoreFixture)fixture[0];
             for (FullGCMode gcType : FullGCMode.values()) {
+                if (gcType == FullGCMode.ORPHANS_EMPTYPROPS_BETWEEN_CHECKPOINTS_NO_UNMERGED_BC
+                        || gcType == FullGCMode.ORPHANS_EMPTYPROPS_BETWEEN_CHECKPOINTS_WITH_UNMERGED_BC) {
+                    // temporarily skip due to flakyness
+                    continue;
+                }
+                if (f.getName().equals("Memory") || f.getName().startsWith("RDB")) {
+                    // then only run NONE and EMPTY_PROPS, cause we are rolling EMPTY_PROPS first
+                    if (gcType != FullGCMode.NONE && gcType != FullGCMode.EMPTYPROPS
+                            && gcType != FullGCMode.GAP_ORPHANS_EMPTYPROPS) {
+                        // temporarily skip due to slowness
+                        continue;
+                    }
+                }
```

Comment:
is this necessary? I'm aware we have the same over in VersionGarbageCollectorIT - but that was introduced based on some test flakyness. But for BranchCommitGCTest adding this would now disable fully functioning tests.

Reply:
This is not necessary. I will revert the changes.